doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1475,6 +1475,92 @@ The Opcodes Aggregator client is an optional Bluetooth mesh model that can be en
 	Abort the Opcodes Aggregator Sequence message. This command clears the Opcodes Aggregator Client context.
 
 
+Remote Provisioning Client
+--------------------------
+
+The Remote Provisioning Client is an optional Bluetooth mesh model enabled through the :kconfig:option:`CONFIG_BT_MESH_RPR_CLI` configuration option. The Remote Provisioning Client model provides support for remote provisioning of devices into a mesh network by using the Remote Provisioning Server model.
+
+This shell module can be used to trigger interaction between Remote Provisioning Clients and Remote Provisioning Servers on devices in a mesh network.
+
+``mesh models rpr scan <timeout in seconds> [<UUID>]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Must be at least 1 second.
+	* ``UUID``: Device UUID to scan for. If omitted, all devices will be reported.
+
+``mesh models rpr scan-ext <timeout in seconds> <UUID> [<AD-type> ... ]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start the extended scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Valid values from ``@ref BT_MESH_RPR_EXT_SCAN_TIME_MIN`` to ``@ref BT_MESH_RPR_EXT_SCAN_TIME_MAX``.
```

Comment:
Do we need @ref or could it be omitted?

Reply:
I change this to :c:macro: and that will show this in a special way in the documentation.